sparkle/src/styles/tailwind.css

```diff
@@ -2,12 +2,26 @@
 @tailwind components;
 @tailwind utilities;
 
+.s-blinking-cursor> :not(pre):last-child::after {
```

Comment:
noob: for my understanding why do we need to copy that style here and in `sparkle/src/styles/global_with_tw_base.css`?

Reply:
And in `sparkle/src/styles/global.css`

Reply:
`global_with_tw_base` is used for storybook . Not sure about `global.css`, will check

Reply:
global.css is used in rollup